zigzag/cost_model/port_activity.py

```diff
@@ -0,0 +1,83 @@
+from zigzag.datatypes import LayerOperand
+from zigzag.hardware.architecture.memory_port import DataDirection
+
+
+class PortActivity:
+    """!  Class that collects all the data transfer rate (periodic) information for each DTL (data transfer link)."""
```

Comment:
What are all these exclamation marks doing in the comments?

Reply:
https://www.doxygen.nl/manual/docblocks.html#pythonblocks
The other way is setting `PYTHON_DOCSTRING NO` in the configuration.
This is fine for me too though, I just had never seen it before and it doesn't seem very common

Reply:
So the docstring can be without the exclamation mark? 

If so, @RobinGeens could you please update the PR as such?

Reply:
Hmm I can't seem to get this working at all. For most classes/functions, the docstring does not show up in the html, regardless of the exclamation mark or the `PYTHON_DOCSTRING NO` setting. Anyone else want to have a try at this?